taskweaver/ces/environment.py

```diff
@@ -230,31 +218,29 @@ def start_session(
 
             self._cmd_session_init(session)
             session.kernel_status = "ready"
+        elif self.mode == EnvMode.Container:
+            if platform.system() != "Windows":
+                # change the permission of the session directory and its subdirectories
+                os.chmod(session.session_dir, 0o755)
+                os.chmod(ces_session_dir, 0o755)
+                os.chmod(cwd, 0o755)
 
-        elif self.mode == EnvMode.OutsideContainer:
-            session = self._get_session(session_id, session_dir=session_dir)
-            ces_session_dir = os.path.join(session.session_dir, "ces")
-            new_kernel_id = get_id(prefix="knl")
-            session.kernel_id = new_kernel_id
-            os.makedirs(ces_session_dir, exist_ok=True)
-            cwd = cwd if cwd is not None else os.path.join(session.session_dir, "cwd")
-            os.makedirs(cwd, exist_ok=True)
             connection_file = self._get_connection_file(session_id, new_kernel_id)
             new_port_start = self.port_start_inside_container
             kernel_env = {
-                "TASKWEAVER_ENV_ID": self.id,
-                "TASKWEAVER_SESSION_ID": session_id,
-                "TASKWEAVER_KERNEL_ID": new_kernel_id,
-                "TASKWEAVER_ENV_DIR": "/app",
```

Comment:
These few variable are still referred in the executor script even in container mode and might could have them reserved as well?

Reply:
will keep some